### ISSUE TITLE
`yarn zip` command not working on Windows

### ISSUE THREAD

**Issue by salexzee** [WorkshopCLI/workshop-advanced-gulp#24]
I've gotten reports this command doesn't work.

**Comment by salexzee** [comment]
More info in this commit https://github.com/WorkshopCLI/workshop-advanced-gulp/commit/c99fa5a8795f8bd6ef2cf810460de0a97acff79a

**Comment by AbWahab07** [comment]
I found a quick fix for that, even though the `yarn zip` doesn't creates a zip folder, if you run the following command 
node -e 'require(\"./workflow/zip/zip.js\").zip()' 
directly in the terminal, it creates the zip folder. I hope we can use this for now, unless there's a solution available.

**Comment by salexzee** [comment]
@AbWahab07 Awesome man! Thanks for looking into this and figuring out this interim fix!

**Comment by salexzee** [comment]
I was finally able to get around to working on this fix in my Windows 10 gaming rig and discovered the issue is related to the escaped double quotes around the relative path.

So I moved the escaped double quotes to wrap around the require(.....).zip() instead and moved the single quotes (unescaped) around the relative path and that fixed the issue in Windows. Tested on my Mac as well. Closing this out.
